src/pages/tokenDrop/Payment.tsx

```diff
@@ -105,6 +107,8 @@ const Payment: FunctionComponent = () => {
   };
 
   const handleCopyToClipboard = async (text: string) => {
+    executeRecaptcha("copyAddress");
```

Comment:
Let's only execute recaptcha on createPurchaseOrder since it's the only one that hits the backend.